keyvault/data-plane/azure-keyvault-extensions/src/main/java/com/microsoft/azure/keyvault/extensions/KeyVaultKey.java

```diff
@@ -38,6 +38,10 @@
 
         @Override
         public byte[] apply(KeyOperationResult result) {
+            if (result == null) {
+                throw new IllegalArgumentException("Parameter 'result' should not be null");
+            }
```

Comment:
You should take a look at the API on the Objects class, as this can be condensed down into a single 'requireNonNull' call.

Reply:
Oh wow. I had no idea this API existed. :D I'll go and update my calls now!